application/features/password.feature

```diff
@@ -3,18 +3,17 @@ Feature: Password
   As an authorized user
   I need to be able to update a specific user password
 
-#  Scenario: Provide a new password for an existing user
-#    Given I receive an existing employee id
-#      And the requestor is authorized
-#      And the user does not already have a password
-#      And I receive a password
-#    When I receive a request to create a password for a specific user
-#    Then a new password hash should be stored
-#      And the last changed date should be stored as the instant it was stored
-#      And the last changed date should be stored in UTC
-#      And the last synched date should be stored as the instant it was stored
-#      And the last synched date should be stored in UTC
```

Comment:
I think this is a decent test to make sure last changed and synced is updated with a new password as I think we use it to determine if a password needs to be changed. Or am I thinking of a different part of the IdP?

Reply:
I think this `password.feature` was a placeholder written well before the system was fully complete. Or at least there are features listed here that are not in the current system. Specifically, the password table doesn't have a "last synced date" nor a "last changed date" column.